doc/Makefile

```
@@ -45,13 +48,17 @@ clean:
 	rm -rf generated/*
 	rm -rf modules/*
 
+sync_git:
+	git clone https://github.com/benchopt/benchmark_logreg_l2 $(BENCH_LOGREG_PATH) > /dev/null 2>&1 || (cd $(BENCH_LOGREG_PATH) ; git pull && cd ../../doc)
```

Maybe just a nitpick: test if the directory exists, and use `pushd/popd` for the git pull.

```suggestion
	if [[ -d !  $(BENCH_LOGREG_PATH) ]]; then
	    git clone https://github.com/benchopt/benchmark_logreg_l2 $(BENCH_LOGREG_PATH) > /dev/null 2>&1
	else
	    pushd $(BENCH_LOGREG_PATH)
	    git pull
	    popd
	fi
```